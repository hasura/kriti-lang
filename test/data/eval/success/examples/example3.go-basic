[
  {{ range _, x := $ }}
   {{ if x.isActive }}
   {{ x.eyeColor }}
   {{ else }}
     "not active"
   {{ end }}
  {{ end }}
]
