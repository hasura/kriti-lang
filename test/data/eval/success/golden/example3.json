[
    [
        "not active",
        "brown",
        "blue",
        "brown",
        "blue"
    ]
]